### PR TITLE
Remainder to Presenter/Requester a week before workshop

### PR DESCRIPTION
**Pull request by lavish205**
#223

### CODE REVIEW

File: wye/workshops/tasks.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from django.core.mail import EmailMessage
+import os
+from wye.workshops.models import Workshop
+
+
+@task
+def workshop_reminder():
+    # date after one week to filter out workshops
+    workshop_date = datetime.date.today() + datetime.timedelta(days=7)
+    workshops = Workshop.objects.filter(expected_date=workshop_date)
```
Comment (vnbang2003) [inline wye/workshops/tasks.py]: Please add condition that workshop needs to be in accepted status.

File: wye/workshops/tasks.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    for workshop in workshops:
+        presenter_email = list()
+        # getting all presenters email
+        presenters = workshop.presenter.all()
+        for presenter in presenters:
```
Comment (vnbang2003) [inline wye/workshops/tasks.py]: We need to even add remainder for requester also.

File: settings/common.py [hunk 3]
```diff
@@ -217,3 +220,17 @@
 TWITTER_CONSUMER_SECRET = os.environ.get("TWITTER_CONSUMER_SECRET", "")
 TWITTER_ACCESS_TOKEN = os.environ.get("TWITTER_ACCESS_TOKEN", "")
 TWITTER_ACCESS_TOKEN_SECRET = os.environ.get("TWITTER_ACCESS_TOKEN_SECRET", "")
+
+# Celery
+djcelery.setup_loader()
+
+BROKER_URL = 'redis://localhost:6379/0'
```
Comment (vnbang2003) [inline settings/common.py]: @vigneshsarma @kracekumar Can you guys confirm the setting ? how do you guys want to handle it

**Comment by lavish205** [comment]
@vnbang2003 please review

**Comment by vnbang2003** [comment]
@lavish205 Code looks good. 
Can you please add test case for it  as coverage had decreased by 1.6% .

**Comment by lavish205** [comment]
@vnbang2003 can you please give me little idea about the test case that I have to write, how we are going to verify that email has been sent?

**Comment by vnbang2003** [comment]
Take a look on test case aded before. Present we cover major of the test cases.
https://github.com/pythonindia/wye/tree/master/tests/functional.
Documentation on how to run local: https://github.com/pythonindia/wye/blob/master/docs/testing.md

**Comment by vnbang2003** [comment]
@lavish205 Any luck with test cases ? If you not able to i will open Bug and accept the merge request for now.

**Pull request merged**